docs/_articles/ja/user-guide/perf-enhancements.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Performance enhancements
+lang: ja
+---
+
+The performance enhancement changes allows you to run single file metadata deploys through a more efficient code path. These changes currently only support Apex Class metadata types. More metadata types will get included in the following releases.
```

Comment:
I think we need the same changes here as were made above ^^

Reply:
Good catch